From 1a1290d54ccabfaa284044c5ce49936c1c44d5ba Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Fri, 19 Jan 2024 19:13:28 +0100
Subject: [PATCH 2/2] Simplify the location of the journal-sched-ext file

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 services/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/meson.build b/services/meson.build
index de640dc..1bccca8 100644
--- a/services/meson.build
+++ b/services/meson.build
@@ -11,6 +11,6 @@ systemd_system_unit_dir = systemd.get_variable(pkgconfig : 'systemdsystemunitdir
     [
       'journald@sched-ext.conf',
     ],
-    install_dir: '/etc/systemd/journald@sched-ext.conf.d'
+    install_dir: '/etc/systemd'
   )
 
-- 
2.43.0.232.ge79552d197

