From 2bdc58f7e2f1396d455aa9e962f276fa25b68941 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Fri, 13 Jun 2025 15:17:20 +0200
Subject: [PATCH 3/3] meson: install scx_mitosis

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 meson-scripts/install_rust_user_scheds | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/meson-scripts/install_rust_user_scheds b/meson-scripts/install_rust_user_scheds
index 30934709..7469215f 100755
--- a/meson-scripts/install_rust_user_scheds
+++ b/meson-scripts/install_rust_user_scheds
@@ -7,12 +7,6 @@ for manifest in "$MESON_SOURCE_ROOT"/scheds/rust/*/Cargo.toml; do
     target_dir="${MESON_BUILD_ROOT}"
     name="${source_dir##*/}"
 
-    # Skip scx_mitosis
-    if [ "$name" = "scx_mitosis" ]; then
-        echo "Skipping installation of $name"
-        continue
-    fi
-
     bins=($(ls -t "${target_dir}/"*"/${name}"))
     if [ ${#bins[@]} -lt 1 ]; then
         echo "Cannot find a binary for $name under $target_dir" 1>&2
-- 
2.49.0

