diff --git a/scheds/rust/meson.build b/scheds/rust/meson.build
index 52b227a..01eeaa3 100644
--- a/scheds/rust/meson.build
+++ b/scheds/rust/meson.build
@@ -1,5 +1,14 @@
+sched = []
+
 subdir('scx_layered')
 subdir('scx_rusty')
 subdir('scx_rustland')
 subdir('scx_rlfifo')
 subdir('scx_lavd')
+
+custom_target('rust_scheds',
+              input: 'meson.build',
+              output: '@PLAINNAME@.__PHONY__',
+              command: ['touch', '@PLAINNAME@.__PHONY__'],
+              build_by_default: true,
+              depends: sched)
diff --git a/scheds/rust/scx_lavd/meson.build b/scheds/rust/scx_lavd/meson.build
index 3604564..1f6ee8a 100644
--- a/scheds/rust/scx_lavd/meson.build
+++ b/scheds/rust/scx_lavd/meson.build
@@ -1,7 +1,8 @@
-custom_target('scx_lavd',
+sched = custom_target('scx_lavd',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              build_by_default: true)
+              build_always_stale: true,
+              depends: [sched])
diff --git a/scheds/rust/scx_layered/meson.build b/scheds/rust/scx_layered/meson.build
index 039f258..f006a10 100644
--- a/scheds/rust/scx_layered/meson.build
+++ b/scheds/rust/scx_layered/meson.build
@@ -1,8 +1,8 @@
-custom_target('scx_layered',
+sched = custom_target('scx_layered',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              depends: [libbpf, bpftool_target],
-              build_by_default: true)
+              build_always_stale: true,
+              depends: [libbpf, bpftool_target, sched])
diff --git a/scheds/rust/scx_rlfifo/meson.build b/scheds/rust/scx_rlfifo/meson.build
index 24d3b64..fd1ffb9 100644
--- a/scheds/rust/scx_rlfifo/meson.build
+++ b/scheds/rust/scx_rlfifo/meson.build
@@ -1,8 +1,8 @@
-custom_target('scx_rlfifo',
+sched = custom_target('scx_rlfifo',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              depends: [libbpf, bpftool_target],
-              build_by_default: true)
+              build_always_stale: true,
+              depends: [libbpf, bpftool_target, sched])
diff --git a/scheds/rust/scx_rustland/meson.build b/scheds/rust/scx_rustland/meson.build
index 9d7c29c..a243771 100644
--- a/scheds/rust/scx_rustland/meson.build
+++ b/scheds/rust/scx_rustland/meson.build
@@ -1,8 +1,8 @@
-custom_target('scx_rustland',
+sched = custom_target('scx_rustland',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              depends: [libbpf, bpftool_target],
-              build_by_default: true)
+              build_always_stale: true,
+              depends: [libbpf, bpftool_target, sched])
diff --git a/scheds/rust/scx_rusty/meson.build b/scheds/rust/scx_rusty/meson.build
index 0510756..b896ce7 100644
--- a/scheds/rust/scx_rusty/meson.build
+++ b/scheds/rust/scx_rusty/meson.build
@@ -1,8 +1,8 @@
-custom_target('scx_rusty',
+sched = custom_target('scx_rusty',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              depends: [libbpf, bpftool_target],
-              build_by_default: true)
+              build_always_stale: true,
+              depends: [libbpf, bpftool_target, sched])
