From 9e6ecaa8fdede16d57a78c539bbee8ba4fe10efc Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Fri, 18 Jul 2025 13:59:52 +0200
Subject: [PATCH 1/7] /etc/default/scx: set scx_flash as default scheduler

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 services/scx | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/services/scx b/services/scx
index f899dfc9..ffac1756 100644
--- a/services/scx
+++ b/services/scx
@@ -1,5 +1,5 @@
-# List of scx_schedulers: scx_bpfland scx_central scx_flash scx_lavd scx_layered scx_nest scx_qmap scx_rlfifo scx_rustland scx_rusty scx_simple scx_userland scx_p2dq scx_tickless
-SCX_SCHEDULER=scx_bpfland
+# List of scx_schedulers: scx_bpfland scx_central scx_flash scx_lavd scx_layered scx_nest scx_p2dq scx_qmap scx_rlfifo scx_rustland scx_rusty scx_simple scx_tickless scx_userland
+SCX_SCHEDULER=scx_flash
 
 # Set custom flags for each scheduler, below is an example of how to use
-#SCX_FLAGS='-p -m performance'
+#SCX_FLAGS='-m performance -D -L'
-- 
2.50.1

