From 5c9bf0e0c6ee1b72478ce08fa05de4a0b543b543 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Tue, 7 May 2024 20:58:03 +0200
Subject: [PATCH 2/5] Exclude hooks to systemd

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 325b882..5cba157 100644
--- a/meson.build
+++ b/meson.build
@@ -330,6 +330,6 @@ endif
 
 libalpm = dependency('libalpm', required: get_option('libalpm'))
 
-if libalpm.found()
+if libalpm.found() and systemd.found()
   subdir('libalpm/hooks')
 endif
-- 
2.44.0.325.g11c821f2f2

